docs/functions.rst

```diff
@@ -57,40 +60,44 @@ however functions which support this alternate API include the line:
+``HIPBLAS_BFLOAT16_CLASS`` before including the header file `hipblas.h`.
+
+There is also an option to interpret the API as using the ``hip_bfloat16`` datatype. This is provided to avoid casting when using the ``hip_bfloat16`` datatype. To expose the API
+using ``hip_bfloat16``, define ``HIPBLAS_USE_HIP_BFLOAT16`` before including the header file `hipblas.h`. 
 
-Furthermore, there is also an option to interpret the API as using the hip_bfloat16 datatype. This is provided to avoid casting when using the hip_bfloat16 datatype. To expose the API
-using hip_bfloat16, HIPBLAS_USE_HIP_BFLOAT16 can be defined before including the header file hipblas.h. Note that the hip_bfloat16 datatype is only supported on AMD platforms.
+.. note::
+    The ``hip_bfloat16`` datatype is only supported on AMD platforms.
 
 Complex Datatypes
 =================
 
-hipBLAS defines hipblasComplex and hipblasDoubleComplex structs. These types contain x and y components and identical memory layout to std::complex
+hipBLAS defines ``hipblasComplex`` and ``hipblasDoubleComplex`` structs. These types contain x and y components and identical memory layout to ``std::complex``
 for float and double precision.
 
-For simplified usage with Hipified code, there is an option to interpret the API as using hipFloatComplex and hipDoubleComplex
-types (i.e. typedef hipFloatComplex hipblasComplex). This is provided for users to avoid casting when using the hip complex types in their code.
-As the memory layout is consistent across all three types, it is safe to cast arguments to API calls between the 3 types: hipFloatComplex,
-std::complex<float>, and hipblasComplex, as well as for the double precision variants. To expose the API as using the hip defined complex types,
-users can use either a compiler define or inline #define ROCM_MATHLIBS_API_USE_HIP_COMPLEX before including the header file <hipblas.h>. Thus, the
+For simplified usage with Hipified code, there is an option to interpret the API as using ``hipComplex`` and ``hipDoubleComplex``
+types (i.e. ``typedef hipComplex hipblasComplex``). This is provided for users to avoid casting when using the hip complex types in their code.
+As the memory layout is consistent across all three types, it is safe to cast arguments to API calls between the 3 types: ``hipComplex``,
+``std::complex<float>``, and ``hipblasComplex``, as well as for the double precision variants. To expose the API as using the hip defined complex types,
+users can use either a compiler define or inline ``#define ROCM_MATHLIBS_API_USE_HIP_COMPLEX`` before including the header file `<hipblas.h>`. Thus, the
 API is compatible with both forms, but recompilation is required to avoid casting if switching to pass in the hip complex types.
 
-Note that hipblasComplex, hipblasDoubleComplex, and use of ROCM_MATHLIBS_API_USE_HIP_COMPLEX are now deprecated. The API will provide interfaces
-using only hipComplex and hipDoubleComplex in the future.
+.. note::
```

Comment:
It should also be noted here that when using the `HIPBLAS_V2` define (as explained above), the API will be exposed with the `hipComplex` and `hipDoubleComplex` datatypes. You can add that here or I can in another PR if you'd rather.

Reply:
> It should also be noted here that when using the `HIPBLAS_V2` define (as explained above), the API will be exposed with the `hipComplex` and `hipDoubleComplex` datatypes. You can add that here or I can in another PR if you'd rather.

I didn't know if hipComplex is preferred over hipFloatComplex?  If so can we only use that form?

Reply:
I suggested opting for hipComplex. I see them both in the HIP documentation so not sure which is preferred. I would assume anyone porting from cuda would expect hipComplex over hipFloatComplex, though.